src/k8s/cmd/k8s/k8s_bootstrap.go

```diff
@@ -65,8 +65,8 @@ func newBootstrapCmd() *cobra.Command {
 		},
 	}
 
-	bootstrapCmd.PersistentFlags().BoolVar(&bootstrapCmdOpts.interactive, "interactive", false, "Interactively configure the most important cluster options.")
-	bootstrapCmd.PersistentFlags().DurationVar(&bootstrapCmdOpts.timeout, "timeout", 90*time.Second, "The max time to wait for k8s to bootstrap.")
+	bootstrapCmd.PersistentFlags().BoolVar(&bootstrapCmdOpts.interactive, "interactive", false, "interactively configure the most important cluster options")
+	bootstrapCmd.PersistentFlags().DurationVar(&bootstrapCmdOpts.timeout, "timeout", 90*time.Second, "the max time to wait for k8s to bootstrap")
```

Comment:
```suggestion
	bootstrapCmd.PersistentFlags().DurationVar(&bootstrapCmdOpts.timeout, "timeout", 90*time.Second, "the maximum time to wait for k8s to bootstrap in seconds")
```

Reply:
Should we add a time expectation here? Is the default time shown to the user?

Reply:
Is it normal to use abbreviation, such as max?

Reply:
> Should we add a time expectation here? Is the default time shown to the user?

This greatly depends on your machine and the current workloads.
e.g. it takes ~5s for on a idle machine but ~30s? when running something heavy in the background.

Reply:
That is true.